docs/astro.config.mjs

```diff
@@ -43,6 +43,7 @@ export default defineConfig({
     "/delta-faq.html": "/delta-faq",
     "/optimizations-oss.html": "/optimizations-oss",
     "/delta-trino-integration.html": "/delta-trino-integration",
+    "/delta-starburst-integration.html": "/delta-starburst-integration",
```

Comment:
No need to add the redirect, this was carryover from the old docs site. You can remove this line.